### DISCUSSION TITLE
Vanity URLs for Craft CDN assets

### DISCUSSION THREAD

**Discussion by matthewstick** [craftcms/cloud#19] [Feature Requests & Enhancements]
Right now all CDN assets go through cdn.craft.cloud

So we have a pretty big traffic on some direct files.  For example, a QR code links to https://domain.com/assets/menu.pdf

When they scan the QR code, it would ideally go to https://domain.com/assets/menu.pdf

But instead it goes to 
https://cdn.craft.cloud/e5c2b9ea-b8f1-4463-86d9-face683786c7/assets/menu.pdf

And now it looks like we've been hacked or a bug.  Because people scan something expecting to see the website, and instead they get your craft cdn.

We could write cusstom URLs and redirects for everything, but it's hard to ask the client to do this.

AWS allows for masked URLs on subpaths.  We do it for most of our sites that have assets folder in S# and regular site in EC2.  But instead of showing theS# url, it appears as the domain name.

Pretty sure you are familiar with this.  Would be great if you could add this.

Thanks!

**Comment by timkelty** [comment]
This is essentially the same issue as https://github.com/craftcms/cloud/discussions/9, so closing as a duplicate
